src/main/java/uk/gov/gchq/palisade/rule/WrappedRule.java

```diff
@@ -43,6 +42,7 @@
  *            by the record reader before being passed to the {@link Rule#apply(Object, User, Context)}.
  */
 public class WrappedRule<T> implements Rule<T> {
+
     public static final String WRAPPED_RULE_WAS_INITIALISED_WITH_NULL = "WrappedRule was initialised with null.";
     public static final String RULE_STRING = "rule";
     public static final String FUNCTION_STRING = "function";
```

Comment:
Use setters in mutator methods

Reply:
lines 67 needs a setter
as does 78
as does 89
and 96, 97 and 98

Reply:
I think this just needs the 2 other getters having the generated annotation adding. I don't think we should have setters as the wrappedRule is immutable.